components/diagnose/diagnose.go

```diff
@@ -170,7 +170,7 @@ func run(ctx context.Context, dir string, opts ...OpOption) error {
 		return err
 	}
 
-	if commandExists("ipmitool") {
+	if process.CommandExists("ipmitool") {
```

Comment:
revert all irrelevant changes. the changes should happen only in the dmesg pkg and affected lines in other pkg.